src/css/bundle.scss

```diff
@@ -1364,4 +1364,7 @@ a.inventory-edit.action-icon.roll {
 span.selection-inventory {
     display: flex;
     margin: 0.2em;
+}
+.grid-2-col{
```

Comment:
Please don't keep adding general rules outside of .sr5.

The system should only modify CSS-rules within it's own context or when alterting Foundry or module styling. The last should live in foundry.scss or module.scss. In bundle.scss only general system specific rules should exist. Therefore this rule (and all above it) should be moved into a .sr5 {} parent rule. :)